docs/content/intro/first-content.md

```diff
@@ -1,5 +1,12 @@
 ---
-title: "First Content"
-date: 2018-02-11T16:52:56-05:00
+title: "Athens 101"
+date: 2018-02-11T16:59:56-05:00
 ---
- First Post!
+
+## What is Athens?
+
+Shortly: Athens is a project building on top of vgo (or go1.11+) trying to bring dependencies closer to you so you can count with repeatable builds even at the time when VCS is down.
```

Comment:
s/can count with repeatable builds even at the time/can count on repeatable builds even at a time/